packages/design-system-tailwind-preset/package.json

```diff
@@ -51,11 +51,15 @@
     "@types/jest": "^27.4.1",
     "deepmerge": "^4.2.2",
     "jest": "^27.5.1",
+    "tailwindcss": "^3.4.13",
     "ts-jest": "^27.1.4",
     "typedoc": "^0.24.8",
     "typedoc-plugin-missing-exports": "^2.0.0",
     "typescript": "~5.2.2"
   },
+  "peerDependencies": {
+    "tailwindcss": "^3.4.13"
+  },
```

Comment:
For `peerDependencies`, I prefer to set the version back to `^3.0.0`. This ensures that if someone adds this package to an existing project using an earlier version (e.g., `3.3.10`), there won't be version conflicts.

Reply:
Great point! Updated here https://github.com/MetaMask/metamask-design-system/pull/41/commits/9e2399491f4b98bdd3926d2a8fc715c16f0a4431